lua/nvim-autopairs/completion/handlers.lua

```diff
@@ -12,7 +12,7 @@ M["*"] = function(char, item, bufnr, commit_character)
     local char_before, char_after = utils.text_cusor_line(line, col, 1, 1, false)
 
     if char == '' or char_before == char or char_after == char
-        or (item.data and item.data.funcParensDisabled)
+    or (item.data and type(data) == 'table' and item.data.funcParensDisabled)
```

Comment:
why data and not `item.data` ?

Reply:
With this obvious mistake the next part has never been evaluated :grin: Thanks for keeping your eye on!

Reply:
Thanks for finding the place to make the change, I started to use phpactor and got the error and this help me to at least fix it locally.

Reply:
https://github.com/windwp/nvim-autopairs/issues/274#issuecomment-1264310074 probably it's not a phpactor bug.